hermes-compose.yml

```diff
@@ -0,0 +1,19 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
```

Comment:
Wouldn't be better to set a fixed version of the cp-zookeeper docker image? If the latest version is bumped and is not compatible anymore with khermes the hermes-compose won't work. What do you think?